aws/hhvm1/dummy-task.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
sorry, license header :)

Reply:
Yeah a lot of files are missing those, I'll add them before merging.

This file will end up particularly funny with a license header though :/